components/generic/ContentCard.vue

```diff
@@ -96,6 +107,10 @@
         type: Array,
         default: () => []
       },
+      hitsText: {
+        type: Array,
```

Comment:
This will have to be an object and should default to null if the changes to the search plugin are implemented.